plugins/modules/k8s_rollback.py

```diff
@@ -17,7 +17,7 @@
 description:
   - Use the OpenShift Python client to perform the Rollback.
   - Authenticate using either a config file, certificates, password or token.
-  - Similar to the kubectl rollout undo command.
+  - Similar to the `kubectl rollout undo` command.
```

Comment:
Do backticks work here or do we need that fancy RST thing like `C()` (I don't know RST well enough).

Reply:
Oh right. Damn. I was thinking Markdown and not RST.